themes/base/storyEdit.html

```diff
@@ -76,6 +76,11 @@
 <link rel="stylesheet" href="/node_modules/infusion/src/framework/core/css/fluid.css">
 
 <script src="src/storyTellingServerUI.js"></script>
+<style>
+    .flc-toc-tocContainer {
+        display: none !important;
+    }
+</style>
```

Comment:
Rather than adding a `<style>` element, it may be cleared just to add an it as an inline style directly on toc container. It would also be good to leave a comment with a link to the JIRA to explain why it's hidden.

Reply:
I had tried this, since inline styling has the highest specificity, but turning the TOC preference on and off will make the element show again.